arabic/dream/It was a nice dream.md

```diff
@@ -0,0 +1,3 @@
+كنت أحاول الفرار من بعض المجرمين وبينما أركض تسائلت لماذا لا أواجه هؤلاء الأوغاد فاستدرت فركلت أولهم فطار فاقداً الوعى ثم لكمت الاخر حتى فقد القدرة على الوقوف فاستمررت على هذا المنوال حتى قضيت على أربعة رجال فى نصف دقيقة قبل أن ينادينى أحدهم بأن أسرع
```

Comment:
It looks like this file has been added to a directory that already contained a "page" of the story.  Please create a new directory for each new page to help keep the repository organized.  See the README of this repository for more information.